index.bs

```diff
@@ -2018,6 +2035,37 @@ ISSUE: properly define the "finishing issuance" operation.
 ISSUE: properly define "begin redemption" operation. Consider running the algorithm at report sending time.
 
 1. Return the result of [=forgiving-base64 encoding=] |redeemedBytes|.
+    
+To <dfn>parse lookback windows</dfn> given an [=ordered map=] |map|:
+
+1. Let |lookbackWindows| be a new [=list=].
+1. If |map|["`lookback_windows`"] does not [=map/exist=], return |aggregatableDedupKeys|.
+1. Let |values| be |map|["`lookback_windows`"].
+1. If |values| is not a [=list=], return null.
+1. [=list/iterate|For each=] |value| of |values|:
+    1. If |value| is not an [=ordered map=], return null.
+    1. If |value|["`window`"] does not [=map/exist=] or is not a [=string=], return null.
```

Comment:
I think this check is redundant, as we already provide `null` as the default to `parse an optional...` below, so we can just check "If |window| is null or an error, return null."